qemu_v8.mk

```diff
@@ -466,6 +479,27 @@ xen-create-image: linux buildroot | $(XEN_TMP)
 run: all
 	$(MAKE) run-only
 
+ifeq ($(WITH_SCMI),y)
+$(SCMI_DTBO): $(SCMI_DTSO)
+	dtc -I dts -O dtb -o $(SCMI_DTBO) $(SCMI_DTSO)
+
+$(SCMI_DTB): $(SCMI_DTBO)
+	cd $(BINARIES_PATH) && $(QEMU_BUILD)/aarch64-softmmu/qemu-system-aarch64 \
+		-nographic \
```

Comment:
There's quite a bit of duplication from the `run-only` target. I believe it's safest to keep as many options as possible the same. How about stuffing all these in a variable that can be used here and in the `run-only` target?

Reply:
+1

Reply:
There is 2 diff : 
 ",dumpdtb=qemu_v8.dtb "
$(QEMU_SCMI_ARGS)

let see what I can do to use a common variable